app/helpers/feeder_form_builder.rb

```diff
@@ -33,9 +33,19 @@ def text_field(method, options = {})
     add_blank_action(options)
     add_changed(method, options)
     add_disabled(options)
+    redact_value(method, options)
     super(method, options)
   end
 
+  def redact_value(method, options)
```

Comment:
Ooh nice, a form helper!

Reply:
seemed like the right place to do this? so far so good.